pkgs/development/python-modules/pytorch-metric-learning/default.nix

```
@@ -0,0 +1,45 @@
+{ buildPythonPackage
+, fetchPypi
+, isPy27
+, numpy
+, scikitlearn
+, pytorch
+, torchvision
+, tqdm
+, lib
+}:
+
+buildPythonPackage rec {
+  pname   = "pytorch-metric-learning";
+  version = "0.9.73";
+
+  disabled = isPy27;
```

```suggestion
  disabled = !isPy3k;
```
Both should work fine.

---

Left without the extra negation for now.